DMCompiler/DM/DMExpression.cs

```diff
@@ -69,6 +69,10 @@ public enum IdentifierPushResult {
             throw new CompileAbortException(Location, $"Cannot reference r-value");
         }
 
+        public virtual string EmitNameof(DMObject dmObject, DMProc proc) {
```

Comment:
`EmitNameof()` doesn't seem like a proper name since it's not actually emitting anything.

Reply:
Suggestions?